docs/api/qiskit-ibm-runtime/release-notes.mdx

```diff
@@ -59,7 +59,7 @@ in_page_toc_max_heading_level: 2
 ### Bug Fixes
 
 *   Fixed support for custom scheduling transpiler stages with Qiskit 2.x. ([2153](https://github.com/Qiskit/qiskit-ibm-runtime/pull/2153))
-*   ConvertConditionsToIfOps \<[api/qiskit/1.4/qiskit.transpiler.passes.ConvertConditionsToIfOps>\_\_](/api/qiskit/1.4/qiskit.transpiler.passes.ConvertConditionsToIfOps>__) now correctly runs at all optimization levels of the scheduling plugins for dynamic circuits, when using Qiskit 1.x. ([2154](https://github.com/Qiskit/qiskit-ibm-runtime/pull/2154))
+*   [ConvertConditionsToIfOps](/api/qiskit/1.4/qiskit.transpiler.passes.ConvertConditionsToIfOps) now correctly runs at all optimization levels of the scheduling plugins for dynamic circuits, when using Qiskit 1.x. ([2154](https://github.com/Qiskit/qiskit-ibm-runtime/pull/2154))
```

Comment:
Did you fix this by changing the HTML artifact, or by changing the MDX file directly? If the latter, it will break the next time we regenerate